helmfile/charts/networkpolicy-common/templates/falco/allow-falco-sidekick.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Values.falco.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-falco-sidekick
+  namespace: falco
+spec:
+  policyTypes:
+    - Ingress
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/instance: falco
+      app.kubernetes.io/name: falcosidekick
+  ingress:
+    - from:
+      - podSelector:
+          matchLabels:
+            app: falco
+      ports:
+        - port: 2801
+{{- end }}
```

Comment:
Shouldn't sidekick be able to talk to some stuff, like the user alertmanager?
https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/falco.yaml.gotmpl#L117

Reply:
I have added falco to alertmanager and added user-alertmanager nps